core/commands/files/files.go

```diff
@@ -0,0 +1,707 @@
+package commands
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	gopath "path"
+	"strings"
+
+	cmds "github.com/ipfs/go-ipfs/commands"
+	core "github.com/ipfs/go-ipfs/core"
+	dag "github.com/ipfs/go-ipfs/merkledag"
+	mfs "github.com/ipfs/go-ipfs/mfs"
+	path "github.com/ipfs/go-ipfs/path"
+	ft "github.com/ipfs/go-ipfs/unixfs"
+	u "github.com/ipfs/go-ipfs/util"
+
+	context "github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+)
+
+var log = u.Logger("cmds/files")
+
+var FilesCmd = &cmds.Command{
+	Helptext: cmds.HelpText{
+		Tagline: "Manipulate unixfs files",
```

Comment:
@whyrusleeping will we use unixfs, files and mfs interchangeably? (looking also at the remaining help descriptions for the commands)